net/src/mock_system.rs

```diff
@@ -162,38 +165,38 @@ impl MockSystem {
     /// we received a SendMessage message...
     /// normally this would travel over the network, then
     /// show up as a HandleSend message, fabricate that message && deliver
-    fn priv_handle_send(&mut self, msg: &MessageData) -> NetResult<()> {
+    fn priv_handle_send_message(&mut self, msg: &MessageData) -> NetResult<()> {
         self.priv_send_one(
             &msg.dna_address,
             &msg.to_agent_id,
-            ProtocolWrapper::HandleSend(msg.clone()).into(),
+            JsonProtocol::HandleSendMessage(msg.clone()).into(),
         )?;
         Ok(())
     }
 
     /// we received a SendResult message...
     /// normally this would travel over the network, then
     /// show up as a SendResult message, fabricate that message && deliver
-    fn priv_handle_send_result(&mut self, msg: &MessageData) -> NetResult<()> {
+    fn priv_handle_handle_send_message_result(&mut self, msg: &MessageData) -> NetResult<()> {
```

Comment:
priv_handle_handle ?

Reply:
Yep. Handle the HandleSendMessageResult message.
Would be clearer if we could do `priv_handle_HandleSendMessageResult()` but rustfmt would bitch about naming conventions...

Reply:
Ah, ok! Looked like one of the accidental double up find and replaces that's caught me up sometimes